frontend/packages/ux-editor-v3/src/components/config/Expressions/NewExpressionButton.tsx

```diff
@@ -14,48 +14,45 @@ export interface NewExpressionButtonProps {
 export const NewExpressionButton = ({ options, onAddExpression }: NewExpressionButtonProps) => {
   const [showDropdown, setShowDropdown] = React.useState<boolean>(false);
   const t = useText();
-  const anchorEl = useRef(null);
 
   return (
-    <>
-      <StudioButton
-        aria-expanded={showDropdown}
-        aria-haspopup='menu'
-        color='first'
-        fullWidth
-        icon={<PlusIcon />}
-        onClick={() => setShowDropdown(!showDropdown)}
-        ref={anchorEl}
-        size='small'
-        title={t('right_menu.expressions_add')}
-        variant='secondary'
-      >
-        {t('right_menu.expressions_add')}
-      </StudioButton>
-      <DropdownMenu
-        anchorEl={anchorEl.current}
-        onClose={() => setShowDropdown(false)}
-        open={showDropdown}
-        placement='top'
-        portal
-        size='small'
-      >
-        <DropdownMenu.Content>
-          <DropdownMenu.Group heading={t('right_menu.expressions_property')}>
-            {options.map((o) => (
-              <DropdownMenu.Item
-                key={o}
-                onClick={() => {
-                  setShowDropdown(false);
-                  onAddExpression(o);
-                }}
-              >
-                {expressionPropertyTexts(t)[o]}
-              </DropdownMenu.Item>
-            ))}
-          </DropdownMenu.Group>
-        </DropdownMenu.Content>
-      </DropdownMenu>
-    </>
+    <DropdownMenu
+      onClose={() => setShowDropdown(false)}
```

Comment:
Should we use StudioDropdown? And is it possible to add a test for what codecov complains on?😄